setup.py

```diff
@@ -31,24 +31,22 @@ def run_meson_build():
     sysargs = meson_call.split(" ")
     sysargs = [arg for arg in sysargs if arg != ""]
     p1 = subprocess.run(sysargs, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+    print(p1.stdout.decode())
     setup_log = os.path.join(staging_dir, "setup.log")
     with open(setup_log, "wb") as f:
         f.write(p1.stdout)
     if p1.returncode != 0:
-        with open(setup_log, "r") as f:
-            print(f.read())
         raise OSError(sysargs, f"The meson setup command failed! Check the log at {setup_log} for more information.")
```

Comment:
Could improve the error message slightly by mentioning that it is also printed above. Bonus points with adding a header/footer to `print(p1.stdout)` to clearly delineate which part is from the log. Can do the same with `p2` below.

Reply:
Fair points. Given the imminent switch to meson-py (I can work on it), I would also be OK if one of the maintainers wanted to merge as-is.